spec/factories/client.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :client do
+    source_app { "my_app" }
+    api_key { "RbZHfHtD1h9XZvs4fGPJUgtt" }
```

Comment:
Should this be here?

Reply:
yes, this is just stubbing for testing purposes